modules/appnexusAstBidAdapter.js

```diff
@@ -183,14 +183,18 @@ function newBid(serverBid, rtbBid) {
     cpm: rtbBid.cpm,
     creative_id: rtbBid.creative_id,
```

Comment:
`appnexusAst` will need `bidderCode` and `creativeId` added to pass validation now